testsuite/parse/ast-parse/literal.verona

```diff
@@ -3,15 +3,17 @@
 
 f()
 {
-  let x1 = 00_42;
-  let x2 = 0xc0_FFeE;
-  let x3 = 0b10_10_10;
-  let x4 = 27.18e-1;
-  let x5 = `foo;
-  let x6 = `(wat ℵ \n);
-  let x7 = `(foo\nbar\) baz);
-  let x8 = `(foo \`\( \ ( \\ baz);
-  let x9 = `(foo `(bar) `bar baz);
-  let xa = 1 + 2 + 3;
-  let xb = 1 + (2 + 3);
+  let int1 = 00_42;
+  let int2 = 0xc0_FFeE;
+  let int3 = 0b10_10_10;
+  let float1 = 27.18e-1;
+  let string1 = `foo;
+  let string2 = `(wat ℵ \n);
+  let string3 = `(foo\nbar\) baz);
+  let string4 = `(foo \`\( \ ( \\ baz);
+  let string5 = `(foo `(bar) `bar baz);
+  let precedence1 = 1 + 2 + 3;
+  let precedence2 = 1 + (2 + 3);
+  let object1 = new { x : U64; };
+  let object2 = new Foo { x : U64; };
```

Comment:
Why are there no values for x here?  I would have thought they would be part of the syntax?

Also, what is `new Foo { x : U64 ; };`, a `Foo` with an additional field?

Reply:
> Why are there no values for x here? I would have thought they would be part of the syntax?

Added.

> Also, what is `new Foo { x : U64 ; };`, a `Foo` with an additional field?

The new object literal would be inheriting from some interface `Foo`.